binsync/common/ui/control_panel.py

```diff
@@ -101,6 +107,11 @@ def _init_widgets(self):
         main_layout.addWidget(self.tabView)
         main_layout.addWidget(self._status_bar)
 
+        self.force_push_button = QPushButton()
```

Comment:
Moving this UI instead to the Utils tab so it is not this huge button on the bottom of the control panel